routes/marketdata.js

```diff
@@ -1,8 +1,61 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const firebase = require('firebase');
+const config = require('./config.js');
+const router = express.Router();
 
-router.get('/', function(req, res, next) {
-	res.render('marketdata');
+if (!firebase.apps.length) {
+    firebase.initializeApp(config.config);
+}
+// Creates connection to database.
+const db = firebase.database();
+// Links to head of database.
+const ref = db.ref("live_weller");
+// Links to markets list.
+const marketsRef = ref.child("markets");
+
+router.get('/:marketName', function(req, res, next) {
+    const marketName = req.params.marketName;
+
+    marketsRef.once('value', function(snapshot) {
+        snapshot.forEach(function(childSnapshot) {
+            const childData = childSnapshot.val();
+            
+            let status = "";
```

Comment:
in general, it's good to declare variables as late as possible (or to reframe that, as close to its initial use as possible)